packages/apollo/src/commands/client/__tests__/generate-new.test.ts

```diff
@@ -0,0 +1,268 @@
+// jest.mock("apollo-codegen-core/lib/localfs", () => {
```

Comment:
lets remove this comment and also rename this file to what we want it to be in the final version.